src/Adapters/DatabasePersistentPageIdentifiersRepo.php

```diff
@@ -14,13 +14,13 @@ public function __construct(
 	) {
 	}
 
-	public function savePersistentId( int $pageId, string $persistentId ): void {
+	public function savePersistentIds( array $ids ): void {
```

Comment:
This array syntax is not nice. This is was caused last night's CI failures when one test had `[ a, b ]` instead of `[ a => b ]`.